bin/fuel-convert

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+import argparse
+import importlib
+import pkgutil
+
+from fuel import converters
+
+
+if __name__ == "__main__":
+    dataset_options = [
+        m[1] for m in pkgutil.iter_modules(converters.__path__) if not m[2]]
```

Comment:
Another way of doing this would be to set `__all__` in `fuel.converters`, and then read the options from there directly, `dataset_options = converters.__all__`.